src/Sylius/Bundle/ShopBundle/Resources/views/Checkout/address.html.twig

```diff
@@ -4,7 +4,7 @@
 
 {% block content %}
     {% include '@SyliusShop/Checkout/_header.html.twig' %}
-    {% include '@SyliusShop/Checkout/_steps.html.twig' %}
+    {% include '@SyliusShop/Checkout/_steps.html.twig' with {'orderTotal': order.total} %}
```

Comment:
TWIG CS

Also, I do no think we need to pass it like this, you have access to entire ``order`` in ``_steps.html.twig``, which is useful. Now when someone overrides the steps template, he loses this functionality, which is not cool and a BC break. :)